mappings/net/minecraft/village/TradeOffer.mapping

```diff
@@ -64,6 +69,9 @@ CLASS net/minecraft/unmapped/C_argwryfu net/minecraft/village/TradeOffer
 	METHOD m_jrpzwrcp depleteBuyItems (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_sddaxwyk;)Z
 		ARG 1 firstBuyStack
 		ARG 2 secondBuyStack
+	METHOD m_jwtgudye read (Lnet/minecraft/unmapped/C_bngyzsts;)Lnet/minecraft/unmapped/C_argwryfu;
+		ARG 0 buf
+	METHOD m_jxiehhbs getIgnoreTags ()Z
```

Comment:
```suggestion
	METHOD m_jxiehhbs shouldIgnoreTags ()Z
```

Reply:
If we're moving away from `get_` terminology, maybe `ignoresTags` or even `ignoresNbt` would be more concise.

Reply:
AFAIK we're not, it's just slightly different for boolean getters (is/does/are)